### PR TITLE
Feature/api approval

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Added option to approve a review and revert approval if required. Approval status is also listed for each revision.

### CODE REVIEW

**Comment by praveenkuttappan** [comment]
Pending approval status screenshot
image

**Comment by praveenkuttappan** [comment]
image

**Comment by praveenkuttappan** [comment]
@pakrym I have deployed this change to staging and added you as an approver.

**Comment by pakrym** [comment]
Great start. One thing I'd like us to do is to change the UI so it takes a little bit less space and is a bit less in your face for people just wanting to look at the API.

File: src/dotnet/APIView/APIViewWeb/Models/ReviewRevisionModel.cs
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Models/ReviewRevisionModel.cs:72]: Let's make this an array, often multiple people would approve an API

File: src/dotnet/APIView/APIViewWeb/Account/ApproverRequirementHandler.cs
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Account/ApproverRequirementHandler.cs:13]: `Use a `Hashset<string>`

**Comment by praveenkuttappan** [comment]
> Great start. One thing I'd like us to do is to change the UI so it takes a little bit less space and is a bit less in your face for people just wanting to look at the API.
> 
> For example, we can display approver information using a checkmark next to the username this would tell us both that it's approved and who done it.
> 
> `✔️pakrym`

Updated to 
image

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml:119]: I don't think there is an approval-label style.

File: src/dotnet/APIView/APIViewWeb/Client/css/site.scss
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Client/css/site.scss:552]: There are already a `font-weight-bolder `style and the `text-success`, `text-danger` styles.

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml:105]: Can we use a single toggle handler instead of two separate ones (see ToggleClosed)/

**Review by pakrym — APPROVED** [review]

**Pull request merged**
